tests/clv/models/test_gamma_gamma.py

```diff
@@ -219,16 +275,14 @@ def test_new_customer_spend(self, distribution):
             preds = model.expected_new_customer_spend()
             assert preds.shape == (1, 1000)
             np.testing.assert_allclose(
-                preds.mean(("draw", "chain")), expected_preds_mean, rtol=0.05
+                preds.mean(("draw", "chain")), expected_preds_mean, rtol=0.1
```

Comment:
Instead of using `self.rng` for the prior predictive let's pass a custom `rng` to this test. It should make it order invariant and hopefully the old `rtol` can be kept. I usually tried a couple of seeds and set on an `rtol` that seemed to work okay. Otherwise we would need to investigate if the change in precision is because of some subtle change in behavior of the model.

Reply:
passing new seed allowed to get back to the old rtol